arthur/server.py

```diff
@@ -147,3 +148,39 @@ def tasks(self):
         logger.debug("Tasks registry read")
 
         return result
+
+    @cherrypy.expose
+    @cherrypy.tools.json_out(handler=json_encoder)
+    def task(self, task_id):
+        """Get info about a task"""
+
+        logger.debug("API 'task' method called for task %s", task_id)
```

Comment:
better to use .format

Reply:
Currently `logging` doesn't support calls to `.format` so I rather keep it as is, to be coherent with the rest of the code where we use the same style. See https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles:

```
Logging calls (logger.debug(), logger.info() etc.) only take positional parameters for the actual logging message itself, with keyword parameters used only for determining options for how to handle the actual logging call (e.g. the exc_info keyword parameter to indicate that traceback information should be logged, or the extra keyword parameter to indicate additional contextual information to be added to the log). So you cannot directly make logging calls using str.format() or string.Template syntax, because internally the logging package uses %-formatting to merge the format string and the variable arguments. There would be no changing this while preserving backward compatibility, since all logging calls which are out there in existing code will be using %-format strings.
```

Reply:
yes I know that logging doesn't support format, but a variable can be created using format (e.g., msg = ....format(...)) and then use it in the following way: logger.debug(msg).

The reason concerning the coherence seems reasonable, however given the fact that there is some ongoing dev on Arthur, it would make sense to update these calls. Anyway I'm fine with any decision, even more if decisions are coherent across the project (and not just at repo level).

Reply:
I see it more like that's they way `logging` is used, which is different to how strings should be formatted. Take into account that `logging` formats the data with the parameters you pass to `debug`, `info`, `error`, etc. It doesn't expects a formatted string only.

Anyway, we can think about it and to take a decision, and be consistent along the project.